pkgs/development/php-packages/phan/default.nix

```
@@ -0,0 +1,46 @@
+{
+  fetchurl,
+  lib,
+  makeWrapper,
+  mkDerivation,
+  php,
+}: let
+  php' = php.withExtensions ({
+    enabled,
+    all,
+  }:
+    enabled ++ [all.ast]);
```

That's not very long - maybe keep it on one line? I found the split + indenting a bit confusing.
```suggestion
  php' = php.withExtensions ({ enabled, all }: enabled ++ [all.ast]);
```